tests/suites/test_suite_ssl.function

```diff
@@ -36,10 +36,18 @@ void log_analyzer( void *ctx, int level,
     }
 }
 
+/* Defaults for min/max version tests (used for arguments as well) */
+#define MBEDTLS_MIN_MAX_VERSION_NONE -1
```

Comment:
What about rather MBEDTLS_SSL_MINOR_VERSION_NONE ? This seems more align with the MBEDTLS_SSL_MINOR_VERSION_x defines.

Reply:
I would agree, but then surely this should be in ssl.h with the other defines. Is it ok to do this, when this define is basically only used here? (Honest question)

Reply:
I don't think that we should define this in `ssl.h`. This is only used in the tests and we don't need this int the public interface.

Reply:
I agree with Janos. I don't think we should define this is ssl.h. This is test specific thus should be defined in the tests. If you worry about the namespace maybe that TESTS_SSL_MINOR_VERSION_NONE or TEST_SSL_MINOR_VERSION_NONE could be an alternative.

Reply:
Ok, done.